central/xlsx/reader.go

```diff
@@ -111,7 +111,7 @@ func convertDTOStdTypeFields(rowElement *reflect.Value, xlsxRow []string, header
 
 			err := setStdStructFieldValue(fieldValue, field)
 			if err != nil {
-				return err
+				return fmt.Errorf("error occurred on '%s' column: %w", headerName, err)
```

Comment:
use errors.EdgeX for the better debugging

Reply:
Should I update all the return error type in [central/xlsx](https://github.com/IOTechSystems/go-mod-core-contracts/tree/v2.3-branch/central/xlsx) dir to errors.EdgeX?

Reply:
yeah, it would be easier to figure out the line producing the error in the first begining

Reply:
Done